src/citra_qt/main.cpp

```diff
@@ -283,6 +287,20 @@ void GMainWindow::ConnectMenuEvents() {
     connect(ui.action_Display_Dock_Widget_Headers, &QAction::triggered, this,
             &GMainWindow::OnDisplayTitleBars);
     connect(ui.action_Show_Status_Bar, &QAction::triggered, statusBar(), &QStatusBar::setVisible);
+
+    // Multiplayer
+    connect(ui.action_Start_Room, &QAction::triggered, this, &GMainWindow::OnStartRoom);
+    connect(ui.action_Connect_To_Room, &QAction::triggered, this, &GMainWindow::OnConnectToRoom);
+    connect(ui.action_Chat, &QAction::triggered, this, &GMainWindow::OnChat);
```

Comment:
Not necessary - chat will be contained in the room window which will be opened by either StartRoom or ConnectToRoom foo.